package.json

```diff
@@ -144,5 +145,11 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "dependencies": {
+    "@responsive-ui/hscroll": "file:components/hscroll",
```

Comment:
We should remove this right?

Reply:
i think since i am on windows, i need these dependencies to run storybook, but i'll remove